win_configure/msi_installer/wix/resources/Samplebat.bat

```diff
@@ -0,0 +1,2 @@
+MKDIR c:\wixexample1
```

Comment:
I think we don't need this file.
same for Thumbs.db, wix/temp/*
Also one suggestion: can we rename 'msi_installer' to just 'msi' ?